packages/react-components/src/components/MessageThread.tsx

```diff
@@ -743,7 +745,7 @@ export const MessageThread = (props: MessageThreadProps): JSX.Element => {
   // To rerender the defaultChatMessageRenderer if app running across days(every new day chat time stamp need to be regenerated)
   const defaultChatMessageRenderer = useCallback(
     (messageProps: MessageProps) => {
-      return DefaultChatMessageRenderer(messageProps);
+      return DefaultChatMessageRenderer(messageProps, ids);
```

Comment:
Recommend sending in only what you need:
```suggestion
      return DefaultChatMessageRenderer(messageProps, { messageTimestamp: ids.messageTimestamp });
```